cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -71,3 +73,40 @@ func (r *Reporter) send(spans []*model.Span, process *model.Process) error {
 	}
 	return err
 }
+
+// addAgentTags appends jaeger tags for the agent to every span it sends to the collector.
+func addAgentTags(spans []*model.Span, agentTags []model.KeyValue) []*model.Span {
+	for _, span := range spans {
+		for _, tag := range agentTags{
+			span.Tags = append(span.Tags, tag)
```

Comment:
top-level question for the PR: should these be added to Span tags or Process tags?

Reply:
It might depend. For example in k8s we support deploying agent as a side-car but also as daemon set (1 agent per k8s node). 

I think we would need an additional flag to be able to add it to process tags

Reply:
> I think we would need an additional flag to be able to add it to process tags

Is that the consensus?